test_pull_requests

```diff
@@ -465,15 +502,17 @@ module Hub
     def merge_pull_request(pull_id, repo, settings, merge_pretest_success_url=nil)
       $stderr.puts "  Merging pull request ##{pull_id} for repo '#{repo}'"
       pull_request, bot_comment, comments = test_merge_pull_request(pull_id, repo, settings)
-      build_url = nil
+      content, build_url, base_commit, extended_tests = extract_bot_comment_metadata(bot_comment['body'], settings)
       if merge_pretest_success_url
         build_url = merge_pretest_success_url
-      elsif bot_comment['body'] =~ /^#{settings['test_prefix']} Running \((.*)\)/
-        build_url = $1
       else
-        raise "'#{settings['name']}' pull request not found"
+        if content =~ /Running/
+          content = 'SUCCESS'
+        elsif content =~ /Quick merging/
```

Comment:
I would rather not add new states.  SUCCESS is good enough and pointing to the test that passed.